packages/build/src/plugins_core/blobs_upload/utils.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'node:fs'
+import fsPromises from 'node:fs/promises'
+import path from 'node:path'
+
+import { fdir } from 'fdir'
+
+const metadataPrefix = '$'
+const metadataSuffix = '.json'
```

Comment:
nit: Using capitalised names for static constants would be more in line with the rest of the codebase.

```suggestion
const METADATA_PREFIX = '$'
const METADATA_SUFFIX = '.json'
```